<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>741</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style de base pour la QGroupBox */
QGroupBox {
    border: 4px solid #000000; /* Bordure bleue */
    border-radius: 25px; /* Coins arrondis */
    margin-top: 10px; /* Marge en haut */
}

/* Style du titre de la QGroupBox */
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 3px;
    background-color: #3498db; /* Fond bleu pour le titre */
    color: #ffffff; /* Texte blanc pour le titre */
}

/* Style des QLabel à l'intérieur de la QGroupBox */
QLabel {
    color: #000000; /* Couleur du texte */
}

/* Style des QRadioButton à l'intérieur de la QGroupBox */
QRadioButton {
    color: #333333; /* Couleur du texte */
}

/* Style des QComboBox à l'intérieur de la QGroupBox */
QComboBox {
    background-color: #ffffff; /* Fond blanc */
    color: #333333; /* Couleur du texte */
    border: 1px solid #3498db; /* Bordure bleue */
    border-radius: 3px; /* Coins arrondis */
    padding: 2px; /* Espacement interne */
}

/* Style des QLineEdit à l'intérieur de la QGroupBox */
QLineEdit {
    background-color: #ffffff; /* Fond blanc */
    color: #333333; /* Couleur du texte */
    border: 1px solid #3498db; /* Bordure bleue */
    border-radius: 3px; /* Coins arrondis */
    padding: 2px; /* Espacement interne */
}

/* Style des QSpinBox à l'intérieur de la QGroupBox */
QSpinBox {
    background-color: #ffffff; /* Fond blanc */
    color: #333333; /* Couleur du texte */
    border: 1px solid #3498db; /* Bordure bleue */
    border-radius: 3px; /* Coins arrondis */
    padding: 2px; /* Espacement interne */
}

/* Style général pour les widgets lorsqu'ils sont désactivés */
QWidget:disabled {
    background-color: #f0f0f0; /* Fond gris clair pour les widgets désactivés */
    color: #a0a0a0; /* Couleur du texte grise pour les widgets désactivés */
}</string>
   </property>
   <property name="title">
    <string>AFFICHAGE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>TITRE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>NBRRE ATHLETE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Date_evennement</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>DUREE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_titre">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Entrez votre TITRE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_nbreathlete">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Entrez votre NBRE ATHLETE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_adresse">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Entrez votre adresse </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_duree">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Entrez votre DUREE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Type de sport : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="lineedit_typedesport">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Choisir Le type de Sport</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Natation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Paddling</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Boating</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kayak</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="lineedit_dateevenement">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
